leaderboard/batch/Batch/SqlJobFlow.cs

```diff
@@ -0,0 +1,129 @@
+using System;
+using Microsoft.Azure.WebJobs;
+using Microsoft.Azure.WebJobs.Host;
+using Microsoft.Extensions.Logging;
+using System.Data;
+using System.Data.SqlClient;
+using System.Diagnostics;
+
+namespace Batch
+{
+    public static class SqlJobFlow
+    {
+        private static SqlConnection connection;
+        private static string queryString = "some batch execution sql";
+
+        static SqlJobFlow()
+        {
+            var connectionString = Environment.GetEnvironmentVariable("SqlConnectionString");
+            connection = new SqlConnection(connectionString);
+            connection.Open();
```

Comment:
You might need a try/catch here so you can log error to console if the conn and execute is failing for whatever reason.

Reply:
Thanks. Done.